rust-toolchain

```diff
@@ -0,0 +1 @@
+1.56.0
```

Comment:
?

Reply:
`actions-rs/toolchain@v1` picks ups the version from `rust-toolchain`.
The last this tests ran with was before the release of `1.57.0` so in conversation we picked that one to make sure everything is the same as before the change.
Can probably be updated afterwards.